### PR TITLE
some concepts updates

### PR DESCRIPTION
**Pull request by rosieyohannan**
further updates

### CODE REVIEW

**Review by catherinepope — APPROVED** [review]
Looks good to me! The only tiny tweak is _sidebar_, rather than _side bar_.

File: jekyll/_cci2/concepts.md [hunk 1]
```diff
@@ -11,15 +11,15 @@ version:
...
+This guide introduces some basic concepts to help you understand how CircleCI manages your [CI/CD](https://circleci.com/continuous-integration/#what-is-continuous-integration) pipelines.
 
 * TOC
 {:toc}
 
 ## Projects
 {: #projects }
 
-A CircleCI project shares the name of the associated code repository in your [Version Control System]({{ site.baseurl }}/2.0/gh-bb-integration/) (VCS). Click **Add Project** in the CircleCI application to enter the Projects dashboard. From here you can set up and follow the projects you have access to.
+A CircleCI project shares the name of the associated code repository in your [Version Control System]({{ site.baseurl }}/2.0/gh-bb-integration/) (VCS). Select **Projects** in the CircleCI web app side bar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
```
Comment (catherinepope) [inline jekyll/_cci2/concepts.md:22]:
```suggestion
A CircleCI project shares the name of the associated code repository in your [Version Control System]({{ site.baseurl }}/2.0/gh-bb-integration/) (VCS). Select **Projects** in the CircleCI web app sidebar to enter the projects dashboard. From here you can set up and follow the projects you have access to.
```

**Review by catherinepope — APPROVED** [review]
Looks good to me!

File: jekyll/_cci2/concepts.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/concepts.md:22]: Thanks!

**Pull request merged**